2.Views/WebAppPages/HomePage.cs

```diff
@@ -0,0 +1,32 @@
+﻿using SeleniumTest.Core;
+using SeleniumTest.Core.Interfaces;
+using UIElements.Commons;
+using UIElements.Enums;
+using UIElements.Interfaces;
+using UIElements.Web;
+
+namespace Views.WebAppPages;
+
+public class HomePage
+{
+    private readonly IGenericWebDriver _driver;
+    public readonly string HostUrl = ConfigModel.HostUrl;
+
+    public IClickable LogoutButton =>
+        new Button(
+            "",
+            new Locator(LocatorType.Id, "ctl00_HeaderTopControl1_LinkButtonLogout"),
+            _driver
+        );
+
+    public HomePage(IGenericWebDriver driver)
+    {
+        _driver = driver;
+        _driver.Instance().Navigate().GoToUrl(HostUrl);
```

Comment:
Page objects are not for navigating to a URL, you can create a helpers class that helps you manage all the general actions that can be done on a web driver, like navigating, taking screenshots, running JS code, handling cookies, etc. Please move that logic there